apps/web/src/components/NavArrow.tsx

```diff
@@ -0,0 +1,43 @@
+import { useRouter } from "next/router";
```

Comment:
The filename should be called `NavArrows.tsx` as it's the component exported

Reply:
Done [7d5cc60](https://github.com/Blobscan/blobscan/pull/479/commits/7d5cc602581d4ba6d82569ee6de0bc8577f165fd)